scripts/ci-e2e.sh

```diff
@@ -15,7 +15,7 @@ export CAPM3RELEASEBRANCH="${CAPM3RELEASEBRANCH:-main}"
 # location according to CAPM3(since CAPM3 minor versions are aligned to CAPI
 # minors versions) release branch
 
-if [[ ${CAPM3RELEASEBRANCH} == "release-1.3" ]]  || [[ ${CAPM3RELEASEBRANCH} == "release-1.4" ]]; then
```

Comment:
a bit unsure if we remove this now already since we would have release-1.3 to main(or some release branch) upgrade test also which would still be running, @mboukhalfa @Sunnatillo any comments?